src/BuildingRegistry.Api.BackOffice/Building/BuildingController-CorrectRealization.cs

```diff
@@ -46,6 +49,7 @@ public partial class BuildingController
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
         [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.GeschetstGebouw.DecentraleBijwerker)]
```

Comment:
@ArneD we have multiple policies for buildings. They can be divided into two groups: 
1. geschetst gebouw
2. ingemeten gebouw. 
Not sure which one applies here?